ui/src/login/ValidatedSignInForm.js

```diff
@@ -44,7 +44,7 @@ export default class ValidatedSignInForm extends Component {
       if (response.status === 200) {
         // store token or something like that to browser storage or wherever you want 
         localStorage.setItem('username', response.data.username)
```

Comment:
if i call localStorage action on this line, the test for check history.push toHaveBeenCalled not passed.